ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-pim-sparse-mode.j2

```diff
@@ -1,11 +1,11 @@
 {# IP PIM Sparse Mode Summary #}
-{% if router_pim_sparse_mode is defined and router_pim_sparse_mode is not none %}
+{% if router_pim_sparse_mode is arista.avd.defined %}
 
 ## Router PIM Sparse Mode
 
 ### IP Sparse Mode Information
-{%     if router_pim_sparse_mode.ipv4 is defined and router_pim_sparse_mode.ipv4 is not none %}
-{%         if router_pim_sparse_mode.ipv4.rp_addresses is defined and router_pim_sparse_mode.ipv4.rp_addresses is not none %}
+{%     if router_pim_sparse_mode.ipv4 is arista.avd.defined %}
+{%         if router_pim_sparse_mode.ipv4.rp_addresses is arista.avd.defined %}
 
 ### IP Rendez-vous Information
```

Comment:
Not your mistake. This heading should have 4 #.

Please change it to ```#### IP Rendezvous Information```

Reply:
Please change Rendez-vous to Rendezvous in line 12

Reply:
No worries. Changes made